assets/style/style.css

```diff
@@ -78,3 +78,279 @@ body {
 .go-down-container {
   margin-top: 50px;
 }
+
+.recent-work-container {
+  display: grid;
+  row-gap: 24px;
+  margin-bottom: 50px;
+}
+
+.recent-work-title-container {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+
+.recent-work-title {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 38px;
+  line-height: 70px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.line-work-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+  margin-bottom: 62px;
+}
+
+.line-work {
+  display: flex;
+  width: 55px;
+  height: 5px;
+  border-radius: 10px;
+  background-color: #36b37f;
+}
+
+.single-card-container {
+  display: flex;
+  justify-content: center;
+}
+
+.single-card {
+  width: 350px;
+  border-radius: 10px;
+  background-color: #eff3f1;
+  border: 1px solid #d0d9d4;
+  overflow: hidden;
+  justify-content: center;
+}
+
+.img-single-work {
+  height: 252px;
+}
+
+.infos-container {
+  background-color: #fff;
+  height: 100%;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.name-single-work {
+  display: flex;
+  align-self: center;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 21px;
+  color: #3a4a42;
+}
+
+.name-single-work h1 {
+  margin-bottom: 0;
+}
+
+.stack-single-work-container {
+  display: flex;
+  justify-content: center;
+  gap: 8px;
+}
+
+.stack-single-work-container ul {
+  display: flex;
+  gap: 10px;
+  padding-left: 0;
+}
+
+.stack-single-container li {
+  background-color: #eff3f1;
+  border-radius: 5px;
+  padding: 10px 12px 10px 12px;
+}
+
+.card-stack-single li {
+  justify-content: center;
+  align-content: center;
+}
+
+.card-stack-single a {
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  text-decoration: none;
+  color: #36b37e;
+}
+
+.stack-single-work-container ul li {
+  list-style: none;
+}
+
+.stack a {
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+}
+
+.see-project-button a {
+  text-decoration: none;
+}
+
+.stack-single-work-container ul li a {
+  text-decoration: none;
+}
+
+.see-project-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.see-project-button {
+  background: #36b37f;
+  border-radius: 4px;
+  padding: 12px 12px 12px 12px;
+}
+
+.see-project-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  padding: 0;
+  color: #fff;
+}
+
+.about-me {
+  display: grid;
+  background-image: url(../images/top-right.svg), url(../images/bottom-left-about-me-mobile.svg);
+  background-position: right top, left bottom;
+  background-repeat: no-repeat, no-repeat;
+  margin-bottom: 60px;
+}
```

Comment:
- [ ] Dear @hansderly, Its one of the main requirements to style this section using a flexbox, please use `display: flex` instead of `grid` to style this section.